build_openssl.bat

```diff
@@ -54,9 +54,12 @@ set OPENSSL_TARGETS=out32
 if NOT %STATIC_LIBS% ( 
 set OPENSSL_TARGETS=%OPENSSL_TARGETS%dll
 )
-if NOT %CONFIG_RELEASE% (
-set OPENSSL_TARGETS=%OPENSSL_TARGETS%.dbg
-)
+
+rem This will trigger build fail since libeay32.lib and ssleay32.lig
+rem are not put in this folder in Debug mode
+rem if NOT %CONFIG_RELEASE% (
+rem set OPENSSL_TARGETS=%OPENSSL_TARGETS%.dbg
```

Comment:
So these files are not needed at all? I suppose if you want to debug Mixxx, it is very unlikely you'll need to dig into debugging OpenSSL.

Reply:
The thing is I have no idea how to build openssl in debug mode. And in the .bat script, even you set Debug, it is actually not built in that way. Probably @rryan can solve the problem.

Reply:
Hello. 
I've found this also when trying to compile in debug.
I also fixed it on my other PR #87 .

The solution is the same, comment that, since it is incorrect. It is modifying the target when it does not need to.